elk/training/burns_norm.py

```diff
@@ -0,0 +1,26 @@
+import torch
+from torch import Tensor, nn
+
+
+class BurnsNorm(nn.Module):
+    """Burns et al. style normalization  Minimal changes from the original code."""
+
+    def forward(self, x: Tensor) -> Tensor:
+        assert (
+            x.dim() == 3
+        ), f"the input should have a dimension of 3 not dimension {x.dim()}, \
+        current shape of input x: {x.shape}"
+
+        x_mean: Tensor = x - torch.mean(x, dim=0)
+        if torch.all(x_mean == 0):
```

Comment:
This is kind of nuts. Let's just use `torch.finfo(x.dtype).eps` in the division to prevent NaNs.